pkg/lib/repo/local/migration.go

```diff
@@ -0,0 +1,77 @@
+package local
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"kitops/pkg/output"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+
+	"oras.land/oras-go/v2"
+)
+
+func NeedsMigrate(baseStoragePath string) (bool, error) {
+	paths, err := findStoragePaths(baseStoragePath)
+	if err != nil {
+		return false, err
+	}
+	return len(paths) > 0, nil
+}
+
+func MigrateStorage(ctx context.Context, baseStoragePath string) error {
+	localStores, err := GetAllLocalStores(baseStoragePath)
```

Comment:
This can be a long running operation. Can we have progress bars? or something that indicates progress.

Reply:
Great point, I've added a basic progress bar (and some reusable code if we ever need a simple bar in the future).